client/javascripts/components/map/map.tag

```diff
@@ -1,6 +1,7 @@
+import { addMap } from './map'
```

Comment:
what's the difference between 
`import { addMap } from './map'`
and
`import addMap from './map'` ?

Reply:
In ES6 you can export several functions, that you can import either with `*` or with `{}`.

`import addMap` won't work as there is no `default` export in `'map'`.

Check [this](https://hackernoon.com/import-export-default-require-commandjs-javascript-nodejs-es6-vs-cheatsheet-different-tutorial-example-5a321738b50f)